dbaas_support.go

```diff
@@ -21,11 +21,31 @@ func init() {
 	dbaasInit = enableDBaaSExtension
 }
 
-func enableDBaaSExtension(mgr ctrl.Manager, cfg mainConfig) {
+func enableDBaaSExtension(mgrOpts manager.Options, crunchybridgeAPIURL string) manager.Manager {
+	mgrOpts.NewCache = cache.BuilderWithOptions(cache.Options{
+		SelectorsByObject: cache.SelectorsByObject{
+			&v1.Secret{}: {
+				Label: labels.SelectorFromSet(labels.Set{
+					dbaasoperator.TypeLabelKey: dbaasoperator.TypeLabelValue,
+				}),
+			},
+		},
+	})
+	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), mgrOpts)
```

Comment:
are not we enabling two NewManager one is main.go
`mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), mgrOpts)`  is this will be issue,

Reply:
we'd overwrite it with dbaas one

Reply:
i mean... this one would overwrite the one in main.go